journalpump/senders/google_cloud_logging.py

```diff
@@ -62,6 +71,18 @@ def send_messages(self, *, messages, cursor):
         for message in messages:
             msg_str = message.decode("utf8")
             msg = json.loads(msg_str)
+
+            # This might not measure exactly 256K but should be a good enough approximation to handle this error.
+            # We try truncating the message if it isn't possible then it is skip.
+            if len(message) > GoogleCloudLoggingSender._LOG_ENTRY_QUOTA:
```

Comment:
Accessing a private/protected members from the public interface is not generally desirable.

Make the `LOG_ENTRY_QUOTA` member public (w/o `_`).
```suggestion
            if len(message) > GoogleCloudLoggingSender.LOG_ENTRY_QUOTA:
```
Or, access it with `self`.
```suggestion
            if len(message) > self._LOG_ENTRY_QUOTA:
```